src/main/java/io/percy/appium/metadata/AndroidMetadata.java

```diff
@@ -46,33 +46,36 @@ public Integer deviceScreenHeight() {
     }
 
     public Integer statBarHeight() {
-
         Integer statBar = getStatusBar();
-        if (statBar == null && orientation != null && orientation.toLowerCase().equals("auto")) {
-            statBar = Utils.extractStatusBarHeight(getDisplaySysDump());
-        }
-        if (statBar != null) {
-            return statBar;
+        if (statBar == null) {
+            try {
+              statBar = Utils.extractStatusBarHeight(getDisplaySysDump());
```

Comment:
Cache this since one call takes around 500 ms
When it is `auto` then call original otherwise use cached.